website/templates/project/register.mako

```diff
@@ -6,7 +6,7 @@
 <%def name="title()">Registration of ${node['title']}</%def>
   <div id="registrationMetaDataScope" class="container scripted">
     <div class="row">
-      <div class="span8 col-md-2 columns eight large-8">        
+      <div class="span8 col-md-2 columns eight large-8">
```

Comment:
Several of these styles do not appear to exist in bootstrap 3 (or elsewhere in our CSS), and should be removed.